docs/orchestrating-elastic-stack-applications/apm-server.asciidoc

```diff
@@ -40,9 +40,25 @@ spec:
   count: 1
   elasticsearchRef:
     name: quickstart
+  kibanaRef: <1>
+    name: quickstart
 EOF
 ----
 
+<1> A reference to a Kibana instance is only required for APM Server versions 8.0.0 and later.
+
+NOTE: If Kibana is referenced in the APM Server specification then the following configuration is required in the Kibana specification to install the APM ILM templates:
```

Comment:
Is that true? It is only required if you are using version 8.0.0 and later. So maybe: something along the lines of: "starting with version 8.0.0 the following Kibana configuration is requireed to run APM Server"

Reply:
Yes but my comment was more about the wording you had. It implies that this is required whenever a `kibanaRef` is specified, when in fact we should qualify this with "starting with 8.0.0"

Reply:
@pebrc My comment may have been misleading, as I had also updated the wording, in additional to double checking the requirement.